### PR TITLE
add bulk subscribers creation api

### PR DESCRIPTION
**Pull request by Eazybright**
This PR adds the functionality to create multiple subscribers at once.
Use Case: 
```ruby
payload = {
    'subscribers' => [
        {
            'name' => 'subscriber-1',
            'email' => 'user1@example.com',
            'firstName' => 'test1',
            'lastName' => 'test1',
            'phone' => '08122442244',
            'subscriberId' => 'subscriber-test-1221'
        },
        {
            'name' => 'subscriber2',
            'email' => 'user2@example.com',
            'firstName' => 'test2',
            'lastName' => 'test2',
            'phone' => '0814422334',
            'subscriberId' => 'subscriber-test-9090'
        }
    ]
}
client.create_bulk_subscribers(payload)
```

### CODE REVIEW

**Review by amansaini0003 — DISMISSED** [review]

File: lib/novu/api/subscribers.rb [hunk 1]
```diff
@@ -208,6 +208,31 @@ def mark_subscriber_feed_seen(subscriber_id, body)
...
+      #     @param `subscriberId` [String] A unique identifier for the subscriber, usually correlates to the id the user in your systems.
+      #     @param `avatar` [String(optional)] An http url to the profile image of your subscriber
+			#
+      # @return data [Hash]
+      #   - updated [Array] - If the subscriber was updated
+      #   - created [Array] - Array of objects for the subsribers ID created
+      #   - failed [Array] - In case of an error, this field will contain the error message
+      #   
+      # @return [number] status - The status code. Returns 201 if the subscribers were created successfully.
+			def create_bulk_subscribers(body)
```
Comment (unicodeveloper) [inline lib/novu/api/subscribers.rb:233]: can you rename this to `bulk_create_subscribers`. The other SDKs have it as bulk create. Just for consistency @Eazybright

**Review by unicodeveloper — COMMENTED** [review]
Please it be bulk create subscribers all over. 🙏🏿

**Comment by unicodeveloper** [comment]
Please also resolve the merge conflicts @Eazybright

**Comment by Eazybright** [comment]
> Please also resolve the merge conflicts @Eazybright

done

**Comment by unicodeveloper** [comment]
It will be pushed next week @Eazybright.

**Comment by unicodeveloper** [comment]
On the other hand, if you can figure out how to create a workflow action that can automatically run to release a new version on rubygem when I tag a new release on GitHub, then that would be fantastic!

E.g if I tag `v2.0.0`, a github action should run that authenticates with the package on rubygem and publishes a new release. @Eazybright

**Comment by unicodeveloper** [comment]
The PHP, Kotlin, Java, Python and .NET SDKs automatically publishes when I tag a new release on the GitHub repo.

**Comment by Eazybright** [comment]
> On the other hand, if you can figure out how to create a workflow GitHub action that can automatically run to release a new version on rubygem when I tag a new release on GitHub, then that would be fantastic!
> 
> E.g if I tag `v2.0.0`, a github action should run that authenticates with the package on rubygem and publishes a new release. @Eazybright

alright, i will work on it.

Maybe you can create an issue for it for documentation puerpose.

**Comment by unicodeveloper** [comment]
Raised the issue here #42  @Eazybright. Please feel free to comment on the issue and ask for assignment.

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**